### DISCUSSION TITLE
如何让flatpak libadwaita应用使用fcitx5的设置？

### DISCUSSION THREAD

**Discussion by Aleksanaa** [fcitx/fcitx5#489] [Q&A]
例如fcitx5的主题、字体大小。
org.google.Chrome、com.getpostman.Postman、com.github.gabutakut.gabutdm和com.github.tchx84.Flatseal可以正常工作。
com.belmoussaoui.Decoder、com.mattjakeman.ExtensionManager、com.rafaelmardojai.Blanket、io.github.realmazharhussain.GdmSettings等使用libadwaita的应用都不能正常工作。
已经尝试设置
```
sudo flatpak override --filesystem=$HOME/.config/fcitx5
sudo flatpak override --filesystem=$HOME/.local/share/fcitx5
```

**Comment by Aleksanaa** [comment]
补充：使用[kimpanel](https://extensions.gnome.org/extension/261/kimpanel/)插件可以解决统一的问题，但是依然无法自定义这些设置

**Reply by wengxt** [reply]
kimpanel 的字体设置在 kimpanel 的 extension 里面

区别主要是是否是原生 wayland 的问题，因为 client side input panel 的情况下需要读取 host 的主题文件

我觉得是你的 override 用法不对，以及你的主题文件可能存在 /usr/share ？

举个例子，我安装在 user 的 org.gnome.gedit ，在这样处理之后，就可以读取到 host 的主题了
flatpak override --filesystem=xdg-config/fcitx5 --filesystem=xdg-data/fcitx5 --user org.gnome.gedit

![image](https://user-images.githubusercontent.com/259684/166491826-ad3bc1a1-fb52-4ba9-bb7c-ed46021fb1e4.png)

**Reply by Aleksanaa** [reply]
> kimpanel 的字体设置在 kimpanel 的 extension 里面
> 
> 区别主要是是否是原生 wayland 的问题，因为 client side input panel 的情况下需要读取 host 的主题文件
> 
> 我觉得是你的 override 用法不对，以及你的主题文件可能存在 /usr/share ？
> 
> 举个例子，我安装在 user 的 org.gnome.gedit ，在这样处理之后，就可以读取到 host 的主题了 flatpak override --filesystem=xdg-config/fcitx5 --filesystem=xdg-data/fcitx5 --user org.gnome.gedit
> 
> ![image](https://user-images.githubusercontent.com/259684/166491826-ad3bc1a1-fb52-4ba9-bb7c-ed46021fb1e4.png)

好像是这样的。谢谢啦！
